build/scripts/windows-exec-sign/generate-testing-credentials.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+CURRENT_DIR="$( dirname $( readlink -f ${BASH_SOURCE[0]} ) )"
+LOCAL_DIR="$CURRENT_DIR/../../../local/testing-pfx-cert"
+IMAGE_NAME="testing-credentials"
+
+rm -rf $LOCAL_DIR && mkdir $LOCAL_DIR
+
+docker build -t $IMAGE_NAME "$CURRENT_DIR/."
+
+docker run --rm -v $LOCAL_DIR:/workdir -w /workdir $IMAGE_NAME bash -c '
+# Generate a private key
+openssl genrsa -out private.key 2048
+
+# Generate a self-signed certificate (valid for 365 days)
+openssl req -new -x509 -key private.key -out certificate.crt -days 365 \
+  -subj "/C=US/ST=TestST/L=TestL/O=TestO Org/OU=TestOrg Unit/CN=test.org.site
+
+PFX_PASSPHRASE="TestPassword123"
+PFX_FILE="certificate.pfx"
+
+# Convert to PFX format
+openssl pkcs12 -export -out $PFX_FILE \
+  -inkey private.key \
+  -in certificate.crt \
+  -passout pass:$PFX_PASSPHRASE
+
+# Encode as base64
+base64 $PFX_FILE > certificate_pfx_base64
+'
```

Comment:
Can't we move this to a script that we can mount and run?

Reply:
We can move it for sure. I added it "inline" to have only one script for generating some testing credentials. Another alternative would be having this script only and leave the `docker run ...` for whoever is using it. WDYT?